TwitchPlaysAssembly/Src/Votes.cs

```diff
@@ -73,6 +73,7 @@ public static class Votes
 				validityChecks = new List<Tuple<Func<bool>, string>>
 				{
 					createCheck(() => !TwitchPlaySettings.data.EnableVoteSolve, "Sorry, {0}, votesolving is disabled."),
+					createCheck(() => voteModule.VoteSolving, "Sorry, {0}, that module is already being votesolved."),
```

Comment:
Another field is not needed for this, we already have `voteModule.Solver.AttemptedForcedSolve`.

Reply:
Done